packages/calypso-products/src/main.ts

```diff
@@ -661,6 +660,4 @@ export function planHasJetpackClassicSearch(
  * Determines if a product is supported by the Atomic sites infrastructure.
  */
 export const isAtomicSupportedProduct = ( productSlug: string ): boolean =>
-	isWpComBusinessPlan( productSlug ) ||
-	isWpComEcommercePlan( productSlug ) ||
-	isMarketplaceProduct( { productSlug } );
+	isWpComBusinessPlan( productSlug ) || isWpComEcommercePlan( productSlug );
```

Comment:
Is there a reason for moving `isMarketplaceProduct( { productSlug } )`  to `client/state/purchases/selectors/should-revert-atomic-site-before-deactivation.js`

https://github.com/Automattic/wp-calypso/pull/58964/files#diff-938fac76d925040c0a694bd3439176989f293f1076c8b703bf8dd741698571dcR10

Reply:
As `isMarketplaceProduct` is now moved to a selector, it's now dependent on state, which is initialized with a fetch. I didn't want to introduce the fetch and state handling in the `calypso-products` package, as all the rest of the values there are static.

As the `isAtomicSupportedProduct` function is only used in this one place, I think it'd be better to move the whole function here.